_posts/Getting Started/2020-07-14-create-flink-project-on-streaming-data-platform.md

```diff
@@ -80,17 +74,96 @@ publishing {
-**1.** Navigate to **Analytics > Analytics Projects > *project-name* > Apps**
+**III.** Navigate to **Analytics > Analytics Projects > *project-name* > Apps**
 ![App-UI]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/app.png)
 
-**2.**  Click **Create New App**. Specify the application name, the artifact source, main class, and other configuration information. You also have the chance to create a new Pravega stream for your application.
+**IV.**  Click **Create New App**. Specify the application name, the artifact source, main class, and other configuration information. You also have the chance to create a new Pravega stream for your application.
 ![Create-App]({{site.baseurl}}/assets/heliumjk/images/post/flink-sdp-setup/create-app.png)
 
-##### F. Check Project Status
-**1.** Navigate to **Analytics > Analytics Projects > *project-name***. Use the links at the top of the project dashboard to view Apache Flink clusters, applications, and artifacts associated with the project.
+#####  <span style="color:#0076ce">Option B: Using Helm Chart to deploy the application</span> 
+
+**I.** Install **[Helm CLI](https://helm.sh/docs/intro/install/)** into your local development environment.
+
+Follow [this guide](https://helm.sh/docs/intro/install/) to install Helm. It can be installed either from source, or from pre-built binary releases.
+
+**II.** Generate the **[Helm Chart](https://helm.sh/docs/topics/charts/)** file structure  
+To use Helm Chart, you need to have a collection of files inside a directory. The directory should be named as `charts`. Then refer to [this github repo](https://github.com/pravega/workshop-samples/tree/master/charts), the [Helm Charts documentation](https://helm.sh/docs/topics/charts/), and the following structure to organize your Chart file.  
+
+```
+charts/               # A directory containing any charts upon which this chart depends
+  Chart.yaml          # A YAML file containing information about the chart
+  values.yaml         # The default configuration values for this chart
+  templates/          # A directory of templates that, when combined with values,
+                      # will generate valid Kubernetes manifest files.
+```
+**III.** Add **FlinkCluster** and **application** `yaml` files to the `template` folder.  
+Next, you need to create template `yaml` files to generate manifest files on the Kubernetes cluster, combining with the configuration in `values.yaml`.
+
+The template folder in [workshop-sample repo](https://github.com/pravega/workshop-samples/tree/master/charts/templates) will give you a general idea for developing your template file. Your chart file structure should look similar to the following way.
+
+```
+charts/              
+  Chart.yaml          
+  values.yaml         
+  templates/          
+    FlinkCluster.yaml               # Template file for Flink Cluster
+    YourApplication.yaml            # Template file for your application. It can be multiple applications.
```

Comment:
It is very difficult to tune and troubleshoot Flink applications in which multiple applications share the same Flink cluster. We should highly recommend one application per cluster as the standard. Once the application has been developed and tested in isolation, advanced users can optimize resources by sharing clusters.